src/components/Content/TaskHierarchy/TaskHierarchy.js

```diff
@@ -96,17 +103,12 @@ class TaskHierarchy extends Component {
   }
 
   render() {
-    const { pursuances, currentPursuanceId } = this.props;
+    const { currentPursuanceId, pursuances } = this.props;
     return (
       <div className="content">
         <div id="task-hierarchy">
           <div id="task-hierarchy-title">
-            <h2 id="tasks-title">Tasks:&nbsp;</h2>
-            <h2 id="pursuance-title">
-              {
-                pursuances[currentPursuanceId] && pursuances[currentPursuanceId].name
-              }
-            </h2>
+            <h2 id="tasks-title">{ this.getPursuanceName(pursuances, currentPursuanceId) }</h2>
```

Comment:
@thevaleriemack Looks like `id` should probably be `pursuance-title`, not `tasks-title` :-)